src/main/java/uk/gov/hmcts/reform/civil/service/docmosis/dq/DirectionsQuestionnaireGenerator.java

```diff
@@ -59,6 +60,12 @@
     public CaseDocument generate(CaseData caseData, String authorisation) {
         DocmosisTemplates templateId = TWO_V_ONE.equals(MultiPartyScenario
             .getMultiPartyScenario(caseData)) ? N181_2V1 : N181;
+
+        if (TWO_V_ONE.equals(MultiPartyScenario.getMultiPartyScenario(caseData))
```

Comment:
It is confusing having two checks close to each other around TWO_V_ONE and both altering templateId. Please can we merge the conditions in line 61 and this one, so that we have one block of code that gives templateId

Reply:
Done